drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/AbstractDataManagementStrategy.java

```diff
@@ -98,7 +98,7 @@ protected static FactModelTree getSimpleClassFactModelTree(Class clazz) {
      */
     protected List<String> getPropertiesToHide(ScenarioGridColumn selectedColumn, ScenarioGridModel scenarioGridModel) {
         List<String> toReturn = new ArrayList<>();
-        if (!selectedColumn.isPropertyAssigned()) {
+        if (selectedColumn != null && !selectedColumn.isPropertyAssigned()) {
```

Comment:
@yesamer 
I see two issues there
1) that getPropertiesToHide is meant to need the selected column, so it must throw an exception if selectedColumn is null (because it means there is a bug in the calling code) - if you find a way to enforce that please do it
2) if undo/redo needs to call that method, it means that some needed information are missing from status, so you have to add them there; said differently, when calling undo/redo, status must have all the informations it needs to set the status

Reply:
@gitgabrio  Reverted and fixed the root cause